tests/core/pipeline/test_pipeline_manager.py

```diff
@@ -119,6 +119,16 @@ def test_set_and_get_pipeline():
     assert _TaskManager._get(task_2.id).id == task_2.id
 
 
+def test_is_submittable():
+    assert len(_PipelineManager._get_all()) == 0
+    pipeline_config = Config.configure_pipeline("pipeline", [])
+    pipeline = _PipelineManager._get_or_create(pipeline_config)
+
+    assert len(_PipelineManager._get_all()) == 1
+    assert _PipelineManager._is_submittable(pipeline)
```

Comment:
Can we test using the PipelineId here as well?
Something like `assert _PipelineManager._is_submittable("PIPELINE_pipeline...)`

Reply:
assert _PipelineManager._is_submittable(pipeline.id)